src/@next/Checkbox/Checkbox.tsx

```diff
@@ -76,10 +80,14 @@ export const Checkbox = React.forwardRef<HTMLDivElement, CheckboxProps>(
         aria-disabled={disabled}
         data-error={hasError}
         data-spacing={isPadded}
+        data-in-table={inTable}
       >
-        <StyledRow>
+        <StyledRow data-in-table={inTable}>
```

Comment:
I'm not sure if this sustainable when we have another need to put checkbox in another control, wouldn't it better to put it on IndexTable's checkbox?

Reply:
you're right, I should move this to IndexTable instead